src/views/AdminActivityEdit.vue

```diff
@@ -138,12 +289,23 @@
 <script>
 import { db } from "@/db.js";
 import { mapGetters } from "vuex";
+import Markdown from "vue-markdown";
 
 export default {
-  props: ["slug", "activityName"],
+  components: {Markdown},
+  props: ["slug", "activityName","currentUserSettings"],
   data() {
     return {
-      currentKey: undefined
+      currentKey: undefined,
+      categories: [
```

Comment:
Doesn't have to be in this PR, but ideally the list of categories would be stored/retrieved from firebase instead of hard-wired.

Reply:
whoops, meant to do that but forgot. Fixed!